modules/openapi-generator/src/main/resources/cpp-ue4/model-base-source.mustache

```diff
@@ -8,7 +8,7 @@ namespace {{this}}
 
 bool HttpRetryManager::Tick(float DeltaTime)
 {
-#if ENGINE_MAJOR_VERSION <= 5 && ENGINE_MINOR_VERSION < 3
+#if ENGINE_MAJOR_VERSION <= 5 && ENGINE_MINOR_VERSION <= 3
```

Comment:
This silences the deprecation warning, but it needs to be completed with the full fix: The HttpRetryManager should not be ticking at all, since it no longer needs to call Update.